main.py

```
@@ -24,12 +24,18 @@
 async def load(ctx, extension):
+    if not ctx.author.guild_permissions.administrator:
+        await ctx.respond("你沒有權限使用這個指令！", ephemeral=True)
+        return
     bot.load_extension(f"cog.{extension}")
     await ctx.send(f"📖 {extension} loaded")
 
 
 @bot.command()
 async def unload(ctx, extension):
+    if not ctx.author.guild_permissions.administrator:
+        await ctx.respond("你沒有權限使用這個指令！", ephemeral=True)
```

Ruff (0.14.10)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)